hudi-common/src/main/java/org/apache/hudi/common/util/LocalAvroSchemaCache.java

```diff
@@ -63,10 +59,4 @@ public Integer cacheSchema(Schema schema) {
   public Option<Schema> getSchema(Integer versionId) {
     return Option.ofNullable(this.versionIdToSchema.get(versionId));
   }
-
-  @Override
-  public void close() {
-    this.schemaToVersionId.clear();
```

Comment:
seems not releated with the change?

Reply:
It's part of the cleanup. Generally there seems to be over use of closeable in some new code. This cleanup will make it easier to be consistent with enforcing that all closeables are closed.